presto-docs/src/main/sphinx/release/release-0.277.rst

```diff
@@ -0,0 +1,27 @@
+=============
+Release 0.277
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add UDF :func:`find_first` to find the first array element which matches a predicate.
+* Fix compilation error in common subexpression elimination when duplicate expressions are present.
+* Fix enforcement of query analyzer timeout during planning.
+* Add support to fail query when number of leaf plan nodes exceeds a limit. This can be controlled with ``leaf_node_limit_enabled`` and ``max_leaf_nodes_in_plan`` session properties.
+
+Delta Lake Connector Changes
+____________________________
+* Upgrade Delta Standalone to 0.5.0.
+
+Pinot Connector Changes
+_______________________
+* Add Pinot binary column type.
+* Add Pinot UDF :func:`pinot_binary_decimal_to_double` to transform Pinot binary column to Double.
```

Comment:
\``BINARY\`` column to \``DOUBLE``

does this resolve into a functions page hyperlink.?
I am not able to find an entry for `pinot_binary_decimal_to_double` function in the rst files.

Reply:
It doesn't resolve. There is no documentation for this function, maybe because its connector specific.

Reply:
maybe add something similar to Teradata functions.
https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/functions/teradata.rst